docs/glassfish/distribution/pom.xml

```diff
@@ -0,0 +1,203 @@
+    <name>Eclipse GlassFish Documentation Distribution</name>
+    <description>
+        Combine the Eclipse GlassFish documentation into one bundle
+        for distribution.
+    </description>
+
+    <properties>
+        <maven.site.skip>true</maven.site.skip>
+        <maven.deploy.skip>false</maven.deploy.skip>
+    </properties>
+
+    <build>
+        <defaultGoal>package</defaultGoal>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <version>1.4.1</version>
+                <executions>
+                    <execution>
+                        <id>enforce-versions</id>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <requireJavaVersion>
+                                    <version>[1.8.0,1.9.0)</version>
+                                    <message>You need JDK8 or lower</message>
+                                </requireJavaVersion>
+                            </rules>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>get-dependencies</id>
+                        <phase>generate-sources</phase>
+                        <goals>
+                            <goal>unpack-dependencies</goal>
+                        </goals>
+                        <configuration>
+                            <includeGroupIds>
+                                org.glassfish.docs
+                            </includeGroupIds>
+                            <excludes>META-INF/**</excludes>
+                            <outputDirectory>
+                                ${project.build.directory}/classes
+                            </outputDirectory>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+	</plugins>
+
+	<pluginManagement>
+	    <plugins>
+		<plugin>
+		    <groupId>org.apache.maven.plugins</groupId>
+		    <artifactId>maven-dependency-plugin</artifactId>
+		    <version>3.1.1</version>
+		</plugin>
+	    </plugins>
+	</pluginManagement>
+    </build>
+
+    <dependencies>
+        <!--
+            The following items are the current
+            (under development) versions of the
+            documentation.
+        -->
+        <dependency>
+            <groupId>org.glassfish.docs</groupId>
+            <artifactId>add-on-component-development-guide</artifactId>
+            <version>5.1.1-SNAPSHOT</version>
```

Comment:
This should use ${project.version} instead. (same applies to all the similar dependencies below)